features/rni.feature

```diff
@@ -0,0 +1,202 @@
+# language: nl
+
+Functionaliteit: RNI-deelnemer voor geleverde gegevens
+
+
+  Rule: de RNI-deelnemer gegevens worden geleverd voor elke categorie waarvan ten minste één gegeven gevraagd is
```

Comment:
Hier begrijp ik niets van. Hoezo worden RNI-deelnemer gegevens geleverd voor elke categorie die gevraagd wordt?
Wat is de relatie tussen RNI-deelnemer en de categoieën? Wellicht die uitleg ook toevoegen aan de documentatie in de specificatie.

Reply:
Hier is niks aan te begrijpen. Per categorie kan zijn aangegeven welke RNI-deelnemer die gegevens heeft aangeleverd. That's it. Ik vind dat wanneer iemand daar meer van wil begrijpen, dat die het LO maar moet lezen.

Reply:
Het antwoord op "Hoezo ..." is: omdat in wet BRP staat dat dat moet. Zoiets zet ik liever niet in een feature. Daarom heb ik me beperkt tot het Wat en Hoe.

Reply:
Helaas wordt ik uit het LO ook niet echt wijzer over wat een RNI-deelnemer nu precies is. Ik neem aan dat het een organisatie is die er verantwoordelijk voor is dat er een persoon in de RNI wordt opgenomen. Dus bijv. het IND of de gemeente als een bewoner heeft gemeld dat deze naar het buitenland verhuisd. Klopt dat?

Reply:
ja, maar niet gemeente. zie de lijst van mogelijke rai-deelnemers in https://publicaties.rvig.nl/dsresource?objectid=fb7de86a-82b8-4ef8-adfc-9c0a9eaae479&type=pdf